CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Changelog
 
-## Unreleased
+## 1.64.0
```

Comment:
But if it's a bugfix release, why call it 1.63.1? (I don't mind, just wondering)

Reply:
forl simplicity, at some point, we decided to count only one number upwards. ideally, we'd like to have just a "Core 64", however, rust forces us to use semantic versioning scheme with 3 digits.

Reply:
:+1: